modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -128,8 +128,8 @@ spec:
 <6> Configuration for an input to filter application logs from the `my-namespace` project.
 <7> Configuration for a pipeline to send audit logs to the secure external Elasticsearch instance:
 ** Optional. A name to describe the pipeline.
-** The `inputRefs` is the log type, in this example `audit`.
-** The `outputRefs` is the name of the output to use, in this example `elasticsearch-secure` to forward to the secure Elasticsearch instance and `default` to forward to the internal Elasticsearch instance.
+** The `inputRefs` are the log types that are inputs for this pipeline, such as `audit` in this example.
+** The `outputRefs` are the names of the outputs to use. For example, `elasticsearch-secure` forwards the audit log data to a secure Elasticsearch instance and `default` forwards the data to the internal Elasticsearch instance.
```

Comment:
The `outputRefs` are --> The `outputRefs` parameters are

Or: Names of the outputs to use.

Reply:
- `elasticsearch-secure` forwards --> the `elasticsearch-secure` output forwards
- `default` forwards --> the `default` output forwards

Reply:
Line 137: `my-app-logs` is the specific input for this pipeline.

Remark: my version is, "`my-app-logs` is the specific input for this Pipeline".

Reason behind this observation and suggestion: In the Pipelines section of the document, all the instance of the the term "pipeline" is written as "Pipeline" specially in the call out content of the YAML: https://docs.openshift.com/container-platform/4.7/cicd/pipelines/understanding-openshift-pipelines.html. Please check and incorporate the same style when it comes to pipeline content. You may want to talk/discuss with Preeti and Souvik, if the need be, to get some clarity on this.

Reply:
- Line 138: 'default` is the output for this pipeline.

Follow the suggestion given above for Line 141 and 142 as well.